ladybug/datatype/base.py

```diff
@@ -114,20 +156,19 @@ def is_in_range(self, values, unit=None, raise_exception=False):
                 unit will be assumed.
             raise_exception: Set to True to raise an exception if not in range.
         """
-        self._check_values(values)
+        assert self._is_numeric(values)
```

Comment:
You probably want to return an error message if it returns a False or it will never happen since _is_numeric will raise an Exception?

Reply:
I would say so. If it does raise an exception then assert is redundant.

Reply:
I got rid of the asserts in order to avoid confusion like yours here.